core/dbt/cli/main.py

```diff
@@ -40,11 +40,14 @@ def __init__(
         project: Project = None,
         profile: Profile = None,
         manifest: Manifest = None,
-        callbacks: List[Callable[[EventMsg], None]] = [],
+        callbacks: List[Callable[[EventMsg], None]] = None,
```

Comment:
ah - reminds me of an all-time classic: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
We have all been caught by it at some point!